util/os-runner/Makefile

```diff
@@ -0,0 +1,26 @@
+OS_HOME=$$HOME/school/os
+OS_IMAGE="os-runner-image"
+OS_CONTAINER="os-runner"
+HOSTNAME="os"
+
+.PHONY: docker-image start run clean
+
+docker-image:
+	docker build -t $(OS_IMAGE) .
+
+start:
+	docker run	\
+		--detach	\
+		--name $(OS_CONTAINER)	\
+		--hostname=$(HOSTNAME)	\
+		--mount type=bind,source="$(OS_HOME)/operating-systems",target=/home/student/operating-systems	\
+		--cap-add SYS_ADMIN	\
+		--env "TERM=xterm-256color"	\
+		-it $(OS_IMAGE)
+
+run:
```

Comment:
```suggestion
run: start
```

Reply:
`make run` is used to create a new session attached to the container created with `make start`. Adding start as a dependency will attempt to create a new container each time you run `make run`.
I will include these facts in a `README.md`.